cli/cmd/policy.go

```diff
@@ -124,6 +124,39 @@ To view the LQL query associated with the policy, use the query id shown.
 		Args:    cobra.ExactArgs(1),
 		RunE:    showPolicy,
 	}
+
```

Comment:
@dmurray-lacework , Do we need any integration test updates here?

Reply:
If we disable Hidden we'll have to update the help text tests.

For now I've added an additional integration test for enable/disable